Found cached dataset xnli (/home/lcur1101/.cache/huggingface/datasets/xnli/fr/1.1.0/818164464f9c9fd15776ca8a00423b074344c3e929d00a2c1a84aa5a50c928bd)
task  NLI
self.device  cuda
  0%|          | 0/3 [00:00<?, ?it/s] 33%|███▎      | 1/3 [00:00<00:00,  6.27it/s]100%|██████████| 3/3 [00:00<00:00, 18.00it/s]
/home/lcur1101/.conda/envs/dl2022/lib/python3.10/site-packages/torch/utils/data/dataloader.py:561: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 3, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Traceback (most recent call last):
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 185, in <module>
    acc = pipeline(LM_model, task, prompt_generator)
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 128, in pipeline
    prompts = prompt_gen(sentences)
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 72, in nli_prompt_generator
    premises, hypotheses = zip(*sentences)
ValueError: too many values to unpack (expected 2)
srun: error: r35n1: task 0: Exited with exit code 1
